tests/foreman/api/test_errata.py

```diff
@@ -75,51 +75,11 @@ def custom_repo(module_org, module_lce, module_cv, activation_key):
     )
 
 
-def _install_package(
-    module_org, clients, host_ids, package_name, via_ssh=True, rpm_package_name=None
-):
-    """Install package via SSH CLI if via_ssh is True, otherwise
-    install via http api: PUT /api/v2/hosts/bulk/install_content
-    """
-    if via_ssh:
-        for client in clients:
-            result = client.run(f'yum install -y {package_name}')
-            assert result.status == 0
-            result = client.run(f'rpm -q {package_name}')
-            assert result.status == 0
-    else:
-        entities.Host().install_content(
-            data={
-                'organization_id': module_org.id,
-                'included': {'ids': host_ids},
-                'content_type': 'package',
-                'content': [package_name],
-            }
-        )
-        _validate_package_installed(clients, rpm_package_name)
-
-
-def _validate_package_installed(hosts, package_name, expected_installed=True, timeout=240):
-    """Check whether package was installed on the list of hosts."""
-    for host in hosts:
-        for _ in range(timeout // 15):
-            result = host.run(f'rpm -q {package_name}')
-            if (
-                result.status == 0
-                and expected_installed
-                or result.status != 0
-                and not expected_installed
-            ):
-                break
-            sleep(15)
-        else:
-            pytest.fail(
-                'Package {} was not {} host {}'.format(
-                    package_name,
-                    'installed on' if expected_installed else 'removed from',
-                    host.hostname,
-                )
-            )
+def _install_package(clients, package_name):
+    """Install package via SSH"""
+    for client in clients:
+        assert client.run(f'yum install -y {package_name}').status == 0
+        assert client.run(f'rpm -q {package_name}').status == 0
```

Comment:
why having  a separate func just to install the package, instead we could just assert the yum/dnf command on client in the test itself, wdyt?
```suggestion
assert rhel_contenthost.run(f'yum install -y {package_name}').status == 0
```

Reply:
heck yeah, after the simplification the function does not make much sense

Reply:
thank you!